app/src/main/java/com/pokemonshowdown/app/BattleFieldActivity.java

```diff
@@ -400,4 +436,104 @@ private void selectItem(int position) {
         mDrawerLayout.closeDrawer(mDrawerList);
     }
 
+    public void showErrorAlert(Exception e) {
```

Comment:
Both of these method can go into the other class. I'm phasing out these ErrorAlert methods because we have a central exception handler already